### PR TITLE
Updated master with the latest image for central-dashboard for v0.6

### PR DESCRIPTION
**Pull request by avdaredevil**
^^

---
/cc @jlewi @prodonjs 
/assign @avdaredevil 
/area front-end
/area centraldashboard
/priority p0

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/3466)

### CODE REVIEW

File: kubeflow/common/prototypes/centraldashboard.jsonnet [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // @description centraldashboard Component
 // @shortDescription centraldashboard
 // @param name string Name
-// @optionalParam image string gcr.io/kubeflow-images-public/centraldashboard:v0.5.0 Image for the central dashboard
+// @optionalParam image string gcr.io/kubeflow-images-public/centraldashboard:v20190613-v0.4.0-rc.1-527-g7ca3cfb3 Image for the central dashboard
```
Comment (prodonjs) [inline kubeflow/common/prototypes/centraldashboard.jsonnet]: Can we retag the image that's being pointed to here so it doesn't appear that it's a stepping backwards?

Comment (jlewi) [reply kubeflow/common/prototypes/centraldashboard.jsonnet]: See https://github.com/kubeflow/kubeflow/issues/2997

There are two issues here

1. The tags of the master branch
1. The expression used in the Makefile to get the descriptor; I think we want to switch to using

    ```
   git describe --tags --long
    ```

Comment (avdaredevil) [reply kubeflow/common/prototypes/centraldashboard.jsonnet]: From the command above I am getting: `v0.4.0-rc.1-528-g65c0b34f`. Is this correct?

**Comment by avdaredevil** [comment]
Presubmit failure fix seems to be available @ #3474.

**Comment by jlewi** [comment]
@avdaredevil Can you rebase off master to pull in the fix to the failing tests (#3475). That should fix the presubmit.

**Comment by prodonjs** [comment]
I would imagine this should be updated in the manifests repo as well, right?

https://github.com/kubeflow/manifests/blob/master/common/centraldashboard/base/kustomization.yaml

**Pull request merged**